.github/workflows/unity.yml

```diff
@@ -0,0 +1,249 @@
+name: Draco Unity library CI
+
+on:
+  # Trigger the workflow on pull request,
+  # but only for the main branch
+  pull_request:
+    branches:
+      - main
+  # Also trigger on release created events
+  release:
+    types:
+      - created
+  workflow_dispatch:
+
+
+jobs:
+
+  mac:
+
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: latest-stable
+
+    - uses: seanmiddleditch/gha-setup-ninja@master
```

Comment:
Can you pin this to a tag? It's probably harmless, but my preference is to at least reference a release here. I think v3 will work.